challenges/objects-arrays.js

```diff
@@ -54,12 +88,26 @@ The resulting contact information should have a space between the first name and
 Name email@example.com
 
 Log the result of your new array. */
+
 const contactInfo = [];
+
+for (let i = 0; i <= index ; i++) {
+
+  let nameEmail = `${graduates[i].first_name}${'s'} email is: ${graduates[i].email}`;
+  contactInfo.push(nameEmail);
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
+
 const uni = [];
+for (let i =0; i <= index; i++){
+  if (graduates[i].university.includes('Uni')) {
+    uni.push(graduates[i].university);
+  }
+}
+
```

Comment:
cool. This works. 

Could you use an alternative using regular expressions?? Stretch goal!